GM button: toggle vehicle blockers

Doubleumc opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

# About the pull request

A convenient button to enable/disable those invisible vehicle blockers.

![image](https://github.com/PvE-CMSS13/PvE-CMSS13/assets/14267245/54ab0bdc-d1f3-4cef-bc70-6b799a41e124)
![image](https://github.com/PvE-CMSS13/PvE-CMSS13/assets/14267245/0336c95e-f581-45ea-90d6-f2a0af0aa76d)

<!-- Remove this text and explain what the purpose of your PR is.

Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
If this is an Issue Correction, you can type "Fixes Issue #169420" to link the PR to the corresponding Issue number #169420.

Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying. -->

# Explain why it's good for the game

Significantly more convenient than the existing methods.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding, and may discourage maintainers from reviewing or merging your PR. This section is not strictly required for (non-controversial) fix PRs or backend PRs. -->


# Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>


# Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly label your changes in the changelog. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->
<!-- If you add a name after the ':cl', that name will be used in the changelog. You must add your CKEY after the CL if your GitHub name doesn't match. Maintainers freely reserve the right to remove and add tags should they deem it appropriate. -->

:cl:
add: gm button to toggle vehicle blockers
/:cl:

<!-- Both :cl:'s are required for the changelog to work! -->

morrowwolf reviewed (changes requested):
Instead of having it be a mode toggle I'd far prefer a global list of all vehicle blockers that have an active var that we toggle on/off.

We're confined to a certain number of bitflags and we're starting to walk closer to the limit.

Secondly, and I'm happy to do this if you don't get to it, could you add another section that's just "flags" or "toggles". Should be just changing the categories of this and the other one to "Game Master.Toggles"

Doubleumc commented:
Changed it to work like "Toggle LZ Weeding"/"Toggle OB Spawn" where it toggles a global variable that the effected system checks.

Moved "Toggle Vehicle Blockers" and "Toggle Xeno Spawning" to the Flags section to mirror the Admin tab.

morrowwolf commented on code/modules/admin/admin_verbs.dm:
```diff
@@ -329,6 +329,7 @@ var/list/roundstart_mod_verbs = list(
 		add_verb(src, /client/proc/toggle_game_master)
 		add_verb(src, /client/proc/toggle_join_xeno)
 		add_verb(src, /client/proc/game_master_rename_platoon)
+		add_verb(src, /client/proc/toggle_vehicle_blockers)
```
Also needs removed line 364 for de-admin'ing and such

Doubleumc replied:
Done.

morrowwolf reviewed (dismissed):
Otherwise fine
